VirtoCommerce.Storefront.Model/Tax/TaxRate.cs

```diff
@@ -10,6 +10,7 @@ public TaxRate(Currency currency)
             Rate = new Money(currency);
         }
         public Money Rate { get; set; }
+        public decimal PercentRate { get; set; }
```

Comment:
Why do you need this type change?

Reply:
PercentRate is not money, it's a number. Database field also has decimal instead of money, so I decided to change it